File: Api/Timesheet/Entities/Squad/Squad.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Timesheet.Entities.Attendance
{
    public class Squad : EntityBase
    {
        public string Name { get; set; }
        public string Leader { get; set; }
        public List<string> Members { get; set; }
        public string DefaultArea { get; set; }
        public string DefaultIteration { get; set; }
        public string Manager { get; set; }
        public string KanbanPrefix { get; set; }
        public string KanbanLaneName { get; set; }
        public string KanbanColumnName { get; set; }
        public string ProjectName { get; set; }
        public bool EnablePerformanceAppraisal { get; set; }
        public int? TaskDurationLimit { get; set; } 
    }

}

File: Web/src/app/models/Squad/squad.ts
export interface Squad {
    Id:                         number;
    Name:                       string;
    Leader:                     string;
    Members:                    string[];
    DefaultArea:                string;
    DefaultIteration:           string;
    Manager:                    string;
    KanbanPrefix:               string;
    KanbanLaneName:             string;
    KanbanColumnName:           string;
    ProjectName:                string;
    EnablePerformanceAppraisal: boolean;
    TaskDurationLimit:          number | null;
}

export enum Manager {
    KleytonFazolin = "kleyton.fazolin",
    VictorChuquimia = "victor.chuquimia",
    WesleyMazzo = "wesley.mazzo",
}
